server/config/express.js

```diff
@@ -43,10 +43,16 @@ exports.configure = function configure(app) {
     resave: Boolean(process.env.SESS_RESAVE),
     saveUninitialized: Boolean(process.env.SESS_UNINITIALIZED),
     unset: process.env.SESS_UNSET,
-    cookie: { secure : true },
+    cookie: { httpOnly : true },
     retries: 50
   }));
 
+  // indicate that we are running behind a trust proxy and should use a secure cookie
```

Comment:
There address is local.  There is no configuration necessary, the only thing we need to do is set `trust proxy` to `true` in production and listen on a high number port (like 3000).

We can then put any proxy (Apache, Nginx, HAproxy, etc) on port 80 and 443 of the same machine and route all traffic to localhost:3000 (where the express server is).

For a boolean value, nothing else needs to be configured.